node.js

```diff
@@ -48,7 +48,44 @@ module.exports = {
     'space-before-function-paren': off,
     'keyword-spacing': error,
     'eol-last': [ error, always ],
-    'brace-style': [ error, '1tbs', { allowSingleLine: true } ],
-    'curly': [ error, all ]
+    'brace-style': [ error, '1tbs', { allowSingleLine: false } ],
+    'curly': [ error, all ],
+    'block-spacing': [ error,  always ],
+    'indent': [ error, 2, { SwitchCase: 1 } ],
+    'no-tabs': error,
+    'spaced-comment': [ error, always ],
+    'array-callback-return': warn,
+    'eqeqeq': [ error, always ],
+    'no-else-return': [ warn, { allowElseIf: false } ],
+    'no-await-in-loop': warn,
```

Comment:
off

Reply:
await in loop encourages very poor code imo, and it's always easy to code around with a bit of thought - is there any reason for it to actually be off?

Reply:
in some cases that's true, it's also true that for those cases it's commonly seen mistake, but it's not the case for many cases - anything that requires serial execution or when iteration step depends on the outcome of previous one(s). also for small counts or startup code (ie. code run once) it doesn't matter and code looks way more readable. it's simply not an error.

Reply:
Understood, couldn't those specific cases use `eslint-disable-line` to suppress the warning?

In general application code (especially in APIs) I think it should be warned against (but if you don't like disable-line then I can remove it and separate it into a different discussion/PR)

Reply:
not really, for example this https://github.com/appliedblockchain/bmono/blob/master/examples/erc20-random-balances.js would have to have multiple suppresses... and if it does, what's the point of the rule, then?